en/hardware/pwm_escs_and_servo.md

```diff
@@ -0,0 +1,132 @@
+
+- On a normal fixed wing ESC (which are often abused for multicopters), it provides the output of a BEC to power servos. Its safe to connect to Pixhawk. 
+  Whether its safe to connect **TWO OR MORE** of the +5V outputs to the servo rail depends on the ESC model. 
+  As a rule of thumb only connecting the output of one BEC is a good idea.
+- On a multicopter ESC this line might be not connected at all and neither power the autopilot nor power the ESC if the autopilot applies 5V. 
+  In this case all these can be connected. 
+  Some examples of ESCs built in this fashion are DJI ESCs.
+- On an opto-isolated ESC **without** BEC this line might need to be connected **and powered with 5V from the autopilot**, as it might be needed to drive the opto-isolator. 
+  In this case please provide power to the servo rail by connecting an additional BEC. 
+  It is unsafe to power servos or ESCs from the autopilot's avionics power supply, which is why **Pixhawk series** flight controllersdo not provide power for the servo rail (The AUX servo rail is unpowered and is limited to 1A - always connect an additional BEC).
+
+> **Tip** Instead of cutting the +5V line you can gently lift of the locking tab of the plastic housing of the servo connector for that pin (e.g. using a cutter blade or small screw driver) and pull the pin out. 
+Isolate it with electrical isolation tape and tape it to the servo cable. 
+This allows to undo this later easily.
+
+## PWM Values
+
+The PWM value sent out depends on the flight stack and configuration used. 
+Most ESCs must see a low value before they will spin up a motor to higher values (to protects user who have the throttle stick in the middle position on power-up). 
+This low value barrier varies between roughly 1200 and 900 us. 
+The native stack therefore sends a value of 900 while being disarmed to ensure the ESC sees a valid low pulse. 
+Other flight stacks (e.g. APM) might behave differently.
+
+## Pulse Shape, Voltage and Update Rate
+
+Pixhawk uses active high pulses, like all the major brands (Futaba, Spektrum, FrSky) use. 
+PWM interfaces are not formally standardised, however, the utilised micro controllers use TTL or CMOS voltage levels. 
+TTL is defined as low < 0.8V and high > 2.0V with some manufacturers using > 2.4V for additional noise margin. 
+CMOS logic is defined with similar voltage levels. 
+5V levels are **never** required to successfully switch to an *on* state.
+
+> **Tip** Futaba, FrSky and Spektrum receivers output 3.3V or 3.0V voltage levels, as they are well above 2.4V. 
+  Pixhawk has adopted this common industry pattern and outputs 3.3V levels on recent boards.
+
+Note that on a normal quad rotor motors spin at 100-130 Hz. 
+This means that if the bus operates at 400 Hz the motor receives new commands three to four times during a single revolution, but it is physically impossible to change the blade speed (due to electric effects and blade inertia) that fast.
```

Comment:
This and the next sections can be dropped completely. While this is correct (and the next section partially), it does not really help, because what is relevant is the latency/delay. And a faster output rate helps reducing the latency (that's at least my understanding, and I've also seen this in practice).